src/StreamJsonRpc/JsonMessageFormatter.cs

```diff
@@ -445,6 +526,13 @@ private JsonRpcError ReadError(JToken json)
             };
         }
 
+        private void ClearProgressObject(JToken requestId)
+        {
+            Requires.NotNull(requestId, nameof(requestId));
+
+            this.formatterProgressTracker.ClearProgressMaps(requestId.Value<long>());
+        }
+
         [DebuggerDisplay("{" + nameof(DebuggerDisplay) + ",nq}")]
```

Comment:
At this point, the value of this method is rather low. I suggest you delete it and use the one line that is worth something to replace the one line used by its two callers to call this method.